src/Symfony/Component/Intl/Tests/DateFormatter/AbstractIntlDateFormatterTest.php

```diff
@@ -355,13 +351,9 @@ public function testFormatWithConstructorTimezone()
 
     public function testFormatWithDateTimeZone()
     {
-        if (defined('HHVM_VERSION_ID')) {
-            $this->markTestSkipped('This test cannot work on HHVM. See https://github.com/facebook/hhvm/issues/5875 for the issue.');
-        }
-
-        $formatter = $this->getDateFormatter('en', IntlDateFormatter::MEDIUM, IntlDateFormatter::SHORT, new \DateTimeZone('GMT+03:00'), IntlDateFormatter::GREGORIAN, 'zzzz');
+        $formatter = $this->getDateFormatter('en', IntlDateFormatter::MEDIUM, IntlDateFormatter::SHORT, new \DateTimeZone('GMT'), IntlDateFormatter::GREGORIAN, 'zzzz');
```

Comment:
ping @stof was there a specific reason to test with 'GMT+03:00' here? Because it doesn't work on 5.5.9 (only on 5.5.10+)